src/powershell/Add-EAReader.psm1

```diff
@@ -0,0 +1,54 @@
+<#
+.SYNOPSIS
+Grants EA Reader permissions to the specified Service Principal or Managed Identity
+
+.PARAMETER PrincipalId
+The Object ID of the Service Principal or Managed Identoty.
+
+.PARAMETER PrincipalTenantId
+The Azure Active Directory Tenant which contains the identity.
+
+.PARAMETER BillingAccountId
+The Billing Account ID (Enrollment ID) to grant permissions against.
+
+.EXAMPLE
+Add-EAReader -principalId 00000000-0000-0000-0000-000000000000 -principalTenantId 00000000-0000-0000-0000-000000000000 -billingAccountId 1234567
+
+Grants EA Reader permissions to the specified Service Principal or Managed Identity
+#>
+function Add-EAReader {

```

Comment:
Thinking more broadly, I'd like this to work for all billing scopes in the future. We don't have to do that now, but should we change this to `Add-FinOpsBillingRoleAssignment` and add a `Role` parameter? Also open to better names. We should probably keep it scoped to the FinOps toolkit.

Reply:
Yes.  
My thoughts exactly.  
We need to add support for department and billing account level roles.
With a role parameter and a switch statement to select the correct GUID for the role.